src/encode_xcor.py

```diff
@@ -37,15 +45,48 @@ def parse_arguments():
     log.info(sys.argv)
     return args
 
-def xcor(ta, speak, mito_chr_name, nth, out_dir):
+
+def get_exclusion_range_max(ta, chip_seq_type):
```

Comment:
Is this here just to keep the code in sync with chip?

Reply:
If you see line 70, then `chip_seq_type` is `None` by default. exclusion parameter is activated only when `chip_seq_type` is specified.